modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheValidateIndexes.java

```diff
@@ -230,6 +244,10 @@ public UUID nodeId() {
             String nextArg = argIter.nextArg("");
 
             ValidateIndexesCommandArg arg = CommandArgUtils.of(nextArg, ValidateIndexesCommandArg.class);
+            if (arg == CHECK_CRC) {
```

Comment:
are upper logic : 

> while (argIter.hasNextSubArg() && argsCnt++ < 4) {
still correct ? 
"--cache", "validate_indexes", "--check-crc", "--check-through", "1000", "--check-first" already more than 4, isn`t it ?

Reply:
I've removed the arguments counter because arguments can be more than four.